### PR TITLE
Dev/cgladney/guides banners

### PR DESCRIPTION
**Pull request by Codiegladney-bluescape**
Yet another workaround. This will give custom names to the service swagger files on the left nav. I.e. instead of "staging.collab", it will read "Workspace APIs" on the left side-nav. Sloppy workaround that will likely go away when we have a way of consolidating the swaggers into one larger swagger.

### CODE REVIEW

**Comment by Codiegladney-bluescape** [comment]
Also commented out content of some of the guides pages and put in a banner to state that these APIs will be included at a later date.

File: site-templates/pages/listeners-api.gomd.debughtml [hunk 1]
```diff
@@ -21,19 +35,19 @@
...
     <a href="#listener-states">Listener States</a> <br />
     <a href="#full-example">Full Use Case Example</a> <br />
-    &amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;<a href="#setup-server">1. Setting up a Basic Server</a> <br />
-    &amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;<a href="#setup-proxy">2. Seting Up the Reverse Proxy</a> <br />
-    &amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;<a href="#create-listener">3. Creating a Listener</a> <br />
-    &amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;<a href="#act-on-listener">4. Acting on Listener Events</a></p>
-
-<p></p>
-<br /></p>
+    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#setup-server">1. Setting up a Basic Server</a> <br />
```
Comment (Jackychen-bluescape) [inline site-templates/pages/listeners-api.gomd.debughtml:38]: Do we still need "&nbsp", can these be removed?

**Review by Jackychen-bluescape — APPROVED** [review]
lgtm!

File: site-templates/pages/listeners-api.gomd.debughtml (hunk 1 above)
Comment (Codiegladney-bluescape) [reply site-templates/pages/listeners-api.gomd.debughtml:38]: True. Will remove

**Pull request merged**